lib/PhpStatistics.php

```diff
@@ -112,4 +113,14 @@ protected function getAPCuStatus(): array {
 			'sma' => $smaInfo,
 		];
 	}
+
+	/**
+	 * Get all loaded php extensions
+	 *
+	 * @return string as csv list of loaded extensions
+	 */
+	protected function getLoadedPhpExtensions(): string {
+		$extensions = get_loaded_extensions();
```

Comment:
can you add a check if the function exists or use `@get_loaded_extensions();` ? This sounds like a function that can be disabled on shared hosts.

Reply:
Good point 👍 According to https://www.php.net/manual/en/ini.core.php, any internal PHP function can be disabled via `disable_functions`. Having that in mind would it also make sense to add a `@` to other functions inside this class like `opcache_get_status`, `apcu_cache_info` or `apcu_sma_info`?

Reply:
@ operator will not work.
Please add change like this:
```suggestion
		$extensions = (function_exists('get_loaded_extensions') ? get_loaded_extensions() : ['Unable to list extensions']);
```